model/riscv_step.sail

```diff
@@ -127,7 +127,6 @@ function loop () : unit -> unit = {
 function init_model () -> unit = {
   init_platform (); /* devices */
   init_sys ();      /* processor */
-  init_vmem ();     /* virtual memory */
```

Comment:
Will we not need 'init_vmem()' if we allow a dynamic reset on the Sail model?

Reply:
Added back.